src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,721 @@
+---
+title: "Command - Device Shadow"
+weight: 10
+summary: "Command and control of a device by using the device shadow leveraging MQTT topics"
+---
+
+Command and control is the operation of sending a message to a device requesting it to perform some action or to control the device configuration. However, when interaction happen with devices over intermittent networks or when devices deployed in far-flung areas or when using devices with limited resources, then an IoT solution may not be able to reliably send or receive commands or configuration updates. By using device shadow, IoT applications or services can simulate and control edge device configuration and execute actions.
+
+{{% notice note %}}
+This implementation is designed using the [AWS IoT device shadow service](https://docs.aws.amazon.com/iot/latest/developerguide/iot-device-shadows.html) through reserved MQTT shadow topics, and describes two implementations that use device shadows. Please refer to the [Designing MQTT Topic for AWS IoT Core](https://docs.aws.amazon.com/whitepapers/latest/designing-mqtt-topics-aws-iot-core/designing-mqtt-topics-aws-iot-core.html), specifically the _Applications on AWS_ section. This white paper provides additional alternative patterns that go beyond the scope of this implementation.
+{{% /notice %}}
+
+## Use Cases
+
+- Request or update device state configuration
+  - _I want to track and control smart home devices from my mobile application_
+  - _I want to remotely control household heaters by turning them on/off or by setting up desired temperature_ 
+- Request device actions based on some commands
+  - _I want to remotely unlock a door for a family visitor_ 
+  - _I want to remotely instruct a device for some action, basis of a command_
+
+## Reference Architecture
+
+This section describes architecture considerations for command and control using the AWS IoT Core device shadow
+ - When requesting update to device state configuration
+ - When requesting device actions based on commands  
+
+{{< tabs groupId="MQTT-arch">}}
```

Comment:
The Device Configuration tab is not loaded. Try added a space before the closing `>}}`

Reply:
Working for me. Can you please recheck?